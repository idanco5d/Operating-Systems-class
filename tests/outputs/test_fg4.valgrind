==6291== Memcheck, a memory error detector
==6291== Copyright (C) 2002-2022, and GNU GPL'd, by Julian Seward et al.
==6291== Using Valgrind-3.20.0 and LibVEX; rerun with -h for copyright info
==6291== Command: /mnt/c/Users/illus/os/hw/1/smash
==6291== Parent PID: 6281
==6291== 
==6291== 
==6291== FILE DESCRIPTORS: 4 open (3 std) at exit.
==6291== Open file descriptor 3: /mnt/c/Users/illus/os/hw/1/tests/outputs/test_fg4.valgrind
==6291==    <inherited from parent>
==6291== 
==6291== 
==6291== HEAP SUMMARY:
==6291==     in use at exit: 0 bytes in 0 blocks
==6291==   total heap usage: 135 allocs, 135 frees, 85,640 bytes allocated
==6291== 
==6291== All heap blocks were freed -- no leaks are possible
==6291== 
==6291== For lists of detected and suppressed errors, rerun with: -s
==6291== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
