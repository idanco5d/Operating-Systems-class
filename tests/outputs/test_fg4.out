smash> submitted by Eran
smash> smash> smash> smash> smash> smash> sleep 200& : 9065
smash: got ctrl-Z
smash: process 9065 was stopped
smash> [1] sleep 100& : 9064 2 secs
[2] sleep 200& : 9065 2 secs (stopped)
smash> sleep 100& : 9064
smash: got ctrl-C
smash: process 9064 was killed
smash> [2] sleep 200& : 9065 3 secs (stopped)
smash> smash> sleep 300& : 9223
smash: got ctrl-Z
smash: process 9223 was stopped
smash> [2] sleep 200& : 9065 4 secs (stopped)
[3] sleep 300& : 9223 1 secs (stopped)
smash> smash: sending SIGKILL signal to 2 jobs:
9065: sleep 200&
9223: sleep 300&
